tests/integration/routes/account-test.js

```diff
@@ -209,11 +209,86 @@ getServer(function (error, server) {
     group.end()
   })
 
-  test('PATCH /session/account', {todo: true}, function (t) {
-    // couchdbErrorTests(server, group, couchdbPutUserMock, patchAccountRouteOptions)
-    // invalidTypeErrors(server, group, patchAccountRouteOptions)
+  test('PATCH /session/account', function (group) {
+    var patchAccountRouteOptionsPassword = {
+      method: 'PATCH',
+      url: '/session/account',
+      headers: headersWithAuth,
+      payload: {
+        data: {
+          type: 'account',
+          attributes: {
+            password: 'newsecret'
+          }
+        }
+      }
+    }
+    function patchAccountResponseMock () {
+      return nock('http://localhost:5984')
+        .get('/_users/_design/byId/_view/byId')
+        .query({
+          key: '"abc4567"',
+          include_docs: true
+        })
+        .reply(200, {
+          total_rows: 1,
+          offset: 0,
+          rows: [{
+            doc: {
+              _id: 'org.couchdb.user:pat-doe',
+              _rev: '1-234',
+              password_scheme: 'pbkdf2',
+              iterations: 10,
+              type: 'user',
+              name: 'pat-doe',
+              roles: ['id:userid123', 'mycustomrole'],
+              derived_key: '4b5c9721ab77dd2faf06a36785fd0a30f0bf0d27',
+              salt: 'salt123'
+            }
+          }]
+        })
+        .post('/_users/_bulk_docs', function (body) {
+          return Joi.object({
+            _id: Joi.any().only('org.couchdb.user:pat-doe').required(),
+            _rev: Joi.any().only('1-234').required(),
+            name: Joi.any().only('pat-doe').required(),
+            type: Joi.any().only('user').required(),
+            salt: Joi.string().required(),
+            derived_key: Joi.string().required(),
+            iterations: Joi.any().only(10).required(),
+            password_scheme: Joi.any().only('pbkdf2').required(),
+            roles: Joi.array().items(Joi.string())
+          }).validate(body.docs[0]).error === null
+        })
+        .query(true)
+    }
 
-    t.end()
+    group.test('No Authorization header sent', function (t) {
+      server.inject({
+        method: 'PATCH',
+        url: '/session/account',
+        headers: {}
+      }, function (response) {
+        t.is(response.statusCode, 403, 'returns 403 status')
+        t.end()
+      })
+    })
+
+    group.test('changing password', {only: true}, function (t) {
```

Comment:
@gr2m is this because the code for patch is not written yet so it is returning an error instead of returning the mock response ?    -- when I console log response.result on line 286 I get: 
`{ errors: [ { title: 'Not Found', status: 404, detail: undefined } ] }`